client/src/main/java/au/org/consumerdatastandards/client/model/BankingProductV2.java

```diff
@@ -0,0 +1,269 @@
+/*
+ * Consumer Data Standards
+ * Sample client library to Demonstrate the Consumer Data Right APIs
+ *
+ * NOTE: This class is auto generated by the codegen artefact
+ * https://github.com/ConsumerDataStandardsAustralia/java-artefacts/codegen
+ */
+package au.org.consumerdatastandards.client.model;
+
+import java.time.OffsetDateTime;
+import java.util.List;
+import java.util.Objects;
+
+public class BankingProductV2 implements BankingProduct {
+
+    private String productId;
+
+    private OffsetDateTime effectiveFrom;
+
+    private OffsetDateTime effectiveTo;
+
+    private OffsetDateTime lastUpdated;
+
+    private BankingProductCategory productCategory;
+
+    private String name;
+
+    private String description;
+
+    private String brand;
+
+    private String brandName;
+
+    private String applicationUri;
+
+    private Boolean isTailored;
+
+    private BankingProductAdditionalInformation additionalInformation;
+
+    private List<BankingProductV2CardArt> cardArt;
+
+    /**
+     * A data holder specific unique identifier for this product. This identifier must be unique to a product but does not otherwise need to adhere to ID permanence guidelines.
+     * @return productId
+     */
+    public String getProductId() {
```

Comment:
No @Override annotations in V2 here and others, but they are there in V1...

Reply:
fixed

Reply:
Thanks, you fixed it in this V2 class, but other V2 classes, such as BankingProductV2Details, have the same problem.